### DISCUSSION TITLE
Compatible third party cartridges

### DISCUSSION THREAD

**Discussion by BB-BenBridges** [AxxAxx/AxxSolder#68] [General]
I was wondering if anyone had any experience with cheaper third party cartridges for the C245 handle??

I bought [these](https://www.ebay.com/itm/226119046921) but they just crash the controller when inserted.

Has anyone found some that work?

**Comment by AxxAxx** [comment]
I know a lot of people run the AiXun brand JBC style cartridges. 

But the tips crash the controller? Never seen that happen before.

What happens?

Which firmware are you on? 
How do you power the controller? 
When does it happen?
Did you buy it from me or build it yourself?

If you want to can also text me directly on discord or email at axeljohansson@duck.com.

As I have never seen this before I would like to understand what happens. 

I am currently on vacation so maybe it will take a bit longer to respond than usual..

**Reply by AxxAxx** [reply]
Are you perhaps powering AxxSolder with a USB-PD source while running a firmware lower than 3.1.0 and with a T245 handle? (You can see the fw version if you go into the settings menue, its written on the bottom)

**Reply by BB-BenBridges** [reply]
I'm on v3.0.2. I bought it from you. I powered it via a 12v 2A PSU. Device powers on fine with C245 handle connected and a genuine cartridge connected. If I remove the cartridge and add a third party one, the controller crashes.  A power cycle restarts it but it instantly crashes if the third party cartridge is still installed.
I'll check the brand of tip later as I believe it they were marked even though its not stated on the eBay listing.

**Reply by AxxAxx** [reply]
Okey, 

12V2A, 24W Sounds very low. The T245 tips will try to draw 120 W by default. I am surprised that AxxSolder does not crash with the original tip? But it depends on the capacitance of your PSU..

But probably the non-original tips have slightly lower resistance and therefore the PSU can not provide enough power. 

Do you have a higher power PSU? I would suggest 130-150W 12-24V.

Or you can try to limit the output power from the settings. There should be such a setting even in the old firmware.

//
Axel Johansson

**Reply by BB-BenBridges** [reply]
Not sure what the issue was but the cartridges by OSS Team are working now. Perhaps I had a poor connection somewhere when I last tested.

**Reply by AxxAxx** [reply]
Oh, okey so it is working now?

I have seen something online about that some third party handles and cartridges are slightly different or in some case really bad design compared to the original JBC causing bad connection and short circuit. I guess that third-party cartridges can also differ, if you are lucky they are good, if unlucky - not good.

Anyway if you see it again and think it is something with the controller; ping me and I'll look into it!

**Reply by BB-BenBridges** [reply]
So did a little more testing and it turns out my PSU was too low power. The smaller cartridges with a lower thermal mass worked fine. But the larger ones that required too much power, made the controller turn off. Thinking that may have been my PSU overload cutoff rather than the controller crashing. Anyway, a nice big 24V 10A PSU and all is working.
